parm/config/gfs/config.com

```diff
@@ -38,7 +38,7 @@ if [[ "${RUN_ENVIR:-emc}" == "nco" ]]; then
     COM_OBS_TMPL=$(compath.py "${envir}/obsproc/${obsproc_ver}")'/${RUN}.${YMD}/${HH}/atmos'
     COM_RTOFS_TMPL=$(compath.py "${envir}/${WAVECUR_DID}/${rtofs_ver}")
 else
-    COM_OBS_TMPL='${ROTDIR}/${RUN}.${YMD}/${HH}/obs'
+    COM_OBS_TMPL='${ROTDIR}/${RUN}.${YMD}/${HH}/obs/${MEMDIR}'
```

Comment:
This is going to do this for all ob types, isn't it, is this okay @aerorahul ?

Reply:
Pinging @aerorahul again. Are we okay with creating a `$MEMDIR` for all observations? Will this work as intended out of the box for the atmosphere @jiaruidong2017 ? I don't know if it is preferred to have 80 largely identical sets of directories or 1 directory for most obs and 80 directories with just snow in them?

Reply:
If we want that the `COM_OBS` points only to the `gdas` directory and without `memdir`, we can simply add `MEMDIR=""` where the `COM_OBS` is defined in all other jobs.  An example is below: 

`RUN=${CDUMP} MEMDIR="" YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_OBS:COM_OBS_TMPL
`